nilearn/_utils/helpers.py

```diff
@@ -3,6 +3,47 @@
 import os
 import warnings
 
+OPTIONAL_MATPLOTLIB_MIN_VERSION = "3.3.0"
+
+
+def _set_mpl_backend():
```

Comment:
Please include a one-line docstring

Reply:
Once this is adressed I think we can merge the PR